src/electron/platform/android/setup/steps/prompt-install-service.ts

```diff
@@ -3,13 +3,9 @@
 
 import { AndroidSetupStepConfig } from 'electron/platform/android/setup/android-setup-steps-configs';
 
-export const promptInstallService: AndroidSetupStepConfig = deps => ({
+export const promptInstallService: AndroidSetupStepConfig = stepTransition => ({
     actions: {
-        cancel: () => {
-            deps.stepTransition('prompt-choose-device');
-        },
-        next: () => {
-            deps.stepTransition('installing-service');
-        },
+        cancel: () => stepTransition('prompt-choose-device'),
```

Comment:
You've added cancel and next, but kept the original versions (using deps.stepTransition) on lines 9 and 12. That seems odd since deps is no longer passed in